runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -98,6 +98,91 @@ function M.on_refresh(err, _, ctx, _)
   return vim.NIL
 end
 
+--- @class InlayHintOptions
+--- @field bufnr number|nil
+--- @field filter function|nil
+--- @field id number|nil
+--- @field name string|nil
+--- @field range { start: number[], end: number[] }|nil
+
+--- Returns inlay hints in the (optionally) given buffer, client, and range.
+--- This method does not make an LSP request, but just queries locally stored inlay hints.
+---
+--- @param options table|nil Optional table which holds the following optional fields:
+--- - bufnr (number|nil):
+---     Restrict hints to the clients attached to the given buffer, defaults to the current
+---     buffer (0).
+---
+--- - filter (function|nil):
+---     Predicate used to filter clients. Receives a client as argument and must return a
+---     boolean. Clients matching the predicate are included. Example: <pre>lua
```

Comment:
I tried to use ` ```lua ` but I couldn't make it work with gen_vimdoc.py regardless of what spacing/indentation I tried

Reply:
I think you need to add a blank line before the code block and after it (example [here](https://github.com/MariaSolOs/neovim/blob/3079fa1f9f198bb303fa616004da9c269673c7a5/runtime/lua/vim/_editor.lua#L886-L896)).

Reply:
Yea Markdown code fences are known not to work in "inline" cases like this :(

Reply:
The issue here is the interaction between the code fence and the bullet point. If you put an extra newline, an extra bullet point is inserted